src/components/table/table.scss

```diff
@@ -50,6 +51,10 @@ $table--limel-table--row-selector: 1;
         color: var(--table-text-color);
     }
     .tabulator-tableHolder {
+        padding-bottom: calc(
```

Comment:
https://github.com/Lundalogik/lime-crm-components/assets/35954987/81ba853b-a136-4d30-8977-1f243e648c48

We get this annoying bug where tabulator scrolls to top, every time one of the rows is selected :( 
I've tried to use this as a `margin-bottom`, on the nested container, or as a `margin-bottom` on the last row, as well as other tricks. But nothing works well.

So we should just ignore this, and close the issue. I consider this too hard and not worth fixing.